src/docs/arch-eviction.dox

```diff
@@ -1,15 +1,80 @@
 /*! @arch_page arch-eviction Eviction
 
-Eviction represents the process or removing old data from the cache,
-writing it to disk if it is dirty. The general strategy uses a dedicated
-set of eviction threads that are tasked with identifying candidate data.
-If the data needs to be written, it is reconciled (converting the
-in-memory format to on-disk format), and then written. Clean memory
-can be freed if needed.
+@section eviction_overall Eviction
 
-Eviction cannot be triggered directly by API calls, but happens as a result
-of enough data being dirtied.
+The goal of eviction is to manage the @ref arch-cache and keep the cache in the margin set by the
+user. There is an upper limit for the total content and dirty content, whenever the cache
+goes above the target then eviction gets triggered to bring it down under the limits.
 
 Eviction is managed using \c WT_EVICT_QUEUE structures,
 each of which contains a list of \c WT_EVICT_ENTRY structures.
+
+Eviction is composed of one eviction server, 0 or more eviction worker threads and three
+shared eviction_queues (two ordinary queues and one urgent queue). The eviction server's goal
```

Comment:
"The eviction server's goal is to find the pages that can be evicted and to do so it walks the trees, moves from one
file to the other, and try to find evictable candidates." --> consider rewording:
The eviction server's goal is to find the pages that can be evicted. It has to walk the pages in a tree, one tree after the other, and find evictable candidates.

Reply:
Fixed